app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+    <string name="app_name">IBand Live</string>
+    <string name="iband_token">5Dpf3hEbvuLdXBQs8hK9vjEJEdai8GaGDqne5CkomFXzczBFgopBMts</string>
```

Comment:
Remove token and ast the user to put in in the "getting started" section on the README.